client/profile-wizard/steps/business-details/flows/selective-bundle/selective-extensions-bundle/index.js

```diff
@@ -63,14 +63,63 @@ const installableExtensions = [
 				},
 			},
 			{
-				slug: 'woocommerce-services',
+				slug: 'woocommerce-services:shipping',
```

Comment:
So will there be situations where both of these checkboxes are now displayed but we still just install 1 plugin for the user? I'm a little confused by that, I read the original spec for this and I kinda get it, but I wondered if it would have just been simpler to change the description based on criteria instead, I guess that shifts some complexity elsewhere though so not a requirement to fix, just wanted to raise it to think about.

Reply:
@samueljseay That's a good point. I really wish I did not have to this. This is...ugly and adds unnecessary code as you mentioned in the following comment in the [selective-bundle/index.js](https://github.com/woocommerce/woocommerce-admin/pull/6547/files/8deb4d024fd30df1e5c904b95b7b8e3e35a70728#diff-630468a503ba64957f6143679d5111678c2e989fd26b2a19502163b24ddd41de). 

I think there is at least one case. When the user is in the US, we're supposed to list both WooCommerce Shipping and Tax. The only problem with changing the description is that Shipping and Tax have different copies and they don't fit into one line. I'm sleeping it over night to see if there is a better way 🤔

Reply:
@samueljseay I thought a few ways, but they also require code changes. I think this is okay now with a test added.